#!/usr/bin/env python
PACKAGE = "robocup_object_detector"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('h_th_low', double_t, 0, "Lower limit of h value", 0, 0, 180)
gen.add('h_th_up', double_t, 0, "Lower limit of h value", 180, 0, 180)
gen.add('s_th', double_t, 0, "Lower limit of s value", 0, 0, 255)
gen.add('v_th', double_t, 0, "Lower limit of v value", 0, 0, 255)

exit(gen.generate(PACKAGE, "robocup_object_detector", "bibs_friend"))
